### ISSUE TITLE
How can I change weight decay?

### ISSUE THREAD

**Issue by Xzoky174** [GabrielDornelles/TorchNN-OCR#2]
Hello! I'm new to ML and PyTorch. For a project, I need to adjust the weight decay for the ocr and compare results. How can I do that?

Thank you!

**Comment by GabrielDornelles** [comment]
In Pytorch weight decay (L2 Regularization) is applied in the optimizer. You can change it [here](https://github.com/GabrielDornelles/pytorch-ocr/blob/main/train.py#L109) to:

```py
optimizer = torch.optim.Adam(model.parameters(), lr=cfg.training.lr, weight_decay=0.001)
```

Its not applied currently in the project because I don't think it would change the results, but I would be happy to know the results you achieved.
